modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/extensionflow/MathExtensionTestCase.java

```diff
@@ -0,0 +1,148 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.integration.test.extensionflow;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.databridge.commons.Event;
+import org.wso2.carbon.event.simulator.stub.types.EventDto;
+import org.wso2.carbon.integration.test.client.Wso2EventServer;
+import org.wso2.cep.integration.common.utils.CEPIntegrationTest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Testing Math Ceil Extension
+ */
+public class MathExtensionTestCase extends CEPIntegrationTest {
+	private static final Log log = LogFactory.getLog(MathExtensionTestCase.class);
+
+	@BeforeClass(alwaysRun = true) public void init() throws Exception {
+
+		super.init(TestUserMode.SUPER_TENANT_ADMIN);
+		String loggedInSessionCookie = getSessionCookie();
+
+		eventSimulatorAdminServiceClient = configurationUtil
+				.getEventSimulatorAdminServiceClient(backendURL, loggedInSessionCookie);
+		eventStreamManagerAdminServiceClient = configurationUtil
+				.getEventStreamManagerAdminServiceClient(backendURL, loggedInSessionCookie);
+		eventPublisherAdminServiceClient = configurationUtil
+				.getEventPublisherAdminServiceClient(backendURL, loggedInSessionCookie);
+		eventProcessorAdminServiceClient = configurationUtil
+				.getEventProcessorAdminServiceClient(backendURL, loggedInSessionCookie);
+	}
+
+	@Test(groups = {
+			"wso2.cep" }, description = "Testing Math Ceil Siddhi Extension") public void siddhiMathExtensionTestScenario()
+			throws Exception {
+		final int messageCount = 3;
+
+		int startESCount = eventStreamManagerAdminServiceClient.getEventStreamCount();
+		int startEPCount = eventPublisherAdminServiceClient.getActiveEventPublisherCount();
+		int startEXPCount = eventProcessorAdminServiceClient.getExecutionPlanConfigurationCount();
+
+		//Add StreamDefinition
+		String streamDefinitionAsString1 = getJSONArtifactConfiguration("extensionflows/math",
+		                                                                "org.wso2.event.sensor.stream_1.0.0.json");
+		eventStreamManagerAdminServiceClient.addEventStreamAsString(streamDefinitionAsString1);
+		String streamDefinitionAsString2 = getJSONArtifactConfiguration("extensionflows/math",
+		                                                                "org.wso2.event.sensorClassifyCeil.stream_1.0.0.json");
+		eventStreamManagerAdminServiceClient.addEventStreamAsString(streamDefinitionAsString2);
+		Assert.assertEquals(eventStreamManagerAdminServiceClient.getEventStreamCount(),
+		                    startESCount + 2);
+
+		//Add Execution Plan
+		String executionPlanAsString =
+				getExecutionPlanFromFile("extensionflows/math", "ExecutionPlan.siddhiql");
```

Comment:
@thiliA 
Better provide meaningful names. i.e. if the execution plan name signifies what it does, the event is named as sensorEvent or similar so that the readability is improved.